test/e2e/kubectl/kubectl.go

```diff
@@ -426,17 +426,19 @@ var _ = SIGDescribe("Kubectl client", func() {
 				// ignored for intentionally using standard fields.
 				// This assumption is based on a lack of TableColumnDefinition
 				// in pkg/printers/internalversion/printers.go
-				"ClusterRole": true,
-				"LimitRange":  true,
-				"PodPreset":   true,
+				"ClusterRole":        true,
```

Comment:
Can we nag someone besides us to fix these?

Reply:
this test is not great... there's nothing really wrong with just outputting name/age if there's not a succinct field to summarize

Reply:
added commits addressing the outstanding types that made sense to have custom printers